docs/source/documentation/api_tokens_generation.rst

```diff
@@ -3,7 +3,24 @@ How-to use new API tokens for login
 
 This short guide explains how to manage API tokens in the web application, and use them in the Substra SDK.
 
-.. note::
+.. admonition:: Why generate API tokens?
+
+   The Substra SDK provides a way to log in using username and password (see `substra.Client <references/sdk.html#client>`_).
+   
+   It is safe, but should be used with caution, as it:
+   
+   * doesn't allow for a precise lifetime or separating concerns by creating one token per purpose
+   
+   * may surprise or limite you through its underlying automated session management
+   
+   * can encourage using cleartext passwords, which can end up shared in version control.
```

Comment:
I know that I'm old-school, but I like "semi-colon, semi-colon, point"  after each item of a list. Or any other variant with a typographic sign after each line 😅

Reply:
I wasn't sure about this, I know semicolons is the french style but I don't see it consistently in english documents. I vaguely looked it up and the first link on google was against using any punctuation when one could get away with it. On the other hand [the google style guide](https://developers.google.com/style/lists) uses a point after every list item, which intuitively to me seems excessive. I think we're supposed to follow that